core/driver/sources/src/main/scala/org/virtuslab/ideprobe/reporting/ErrorValidator.scala

```diff
@@ -5,11 +5,14 @@ import org.virtuslab.ideprobe.protocol.IdeMessage
 
 object ErrorValidator {
   def apply(config: CheckConfig, errors: Seq[IdeMessage]): Option[Exception] = {
-    if (errors.isEmpty) None
+    val filteredErrors = errors.filterNot(error => config.ignoredIdeErrors.contains(error.content))
```

Comment:
I'm afraid this might be not powerful enough... we might want to sift out errors where certain parts of the `content` are variable/unknown in advance as well :/